Glossary.md

```diff
@@ -0,0 +1,74 @@
+# Glossary of Terms
+
+## Domain
+
+A specific sphere of activity or knowledge. For example, cyber security, software, bills of material, licensing, software security, etc
+
+## Ontology
+
+An explicit specification of concepts, terms, categories, relationships, etc. for a particular scoped domain forming a language or basis for communication and analysis of information within the scoped domain. SPDX 3.0 is an ontology targeted to support the scoped domain of bills of material.
+
+## Class
+
+A represention of a scope/set of individual instances of a particular “concept” (e.g., File, Person, ExternalReference, etc.). Each individual instance of a class has an IRI ID and is asserted as a member of a particular class via a type statement.
+
+## Domain Class
+
+A class representing a concept of primary focus within a given domain. In a graph or serialized set of instance content this would be the granularity of what would be a node in the graph or a top-level object in the serialization set. This is typically similar to the scoping seen in labeled property graphs.
+
+## Non-domain Class
+
+A class representing a concept of secondary or tertiary focus within a given domain that is a characterization of some aspect of a domain class. When structuring the graph for a domain use case it is desirable and more practical for the domain graph nodes to only consist of domain class instances and object properties. Any non-domain classes and datatype properties would be considered internal adornments on the domain objects.
+
+## Property
+
+A representation of a relationship between a class (definition or instance) and some descriptive characterization of the class.
+
+## Datatype property
+
+A representation of a relationship between a class (definition or instance) and some literal (string, integer, boolean, etc.) descriptive characterization of the class.
+
+## Object property
+
+A representation of a relationship between a class (definition or instance) and some conceptual (class-based) descriptive characterization of the class. In other words, it is a relationship between one class and another class.
+
+## Domain
```

Comment:
Thiere are two definitions of "Domain"

Reply:
that is correct! The second one is definitely an established term ("domain" and "range" of a property). The first one, I assume, is @sbarnum's attempt to talk about "area of interest" or something.
I agree that we should not have both.

Reply:
Can you please update the PR to remove it then.